www/js/services.js

```diff
@@ -224,6 +224,10 @@ app.factory('TodayShCategory', function(){
         return this.news.length;
     }
     
+    TodayShCategory.prototype.getNewsList = function() {
```

Comment:
I think TodaySHCategory and epapercategory object both are the same. We shall refactor to use abstract class.